### PR TITLE
Feature/react components

### PR DESCRIPTION
**Pull request by b2l**
Change the styleguide to make it support react component.

Make sure to install grunt before trying this branch: 
`npm install -g grunt-cli`

You can write component in ES6 or in coffeescript
The documentation for hologram should be as follow:
**ES6:**
```
/*doc
---
title: Title
name: page_name
category: React Components
---

```react_example
// Setup code
var value = 'toto'
--- // <= this line is SUPER important. It's the separation between JSX code and javascript code
<DatePicker value={value} />
```
*/
```

**coffeescript:**
###
/*doc
SAME THING AS FOR THE ES6
###
#*/

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+module.exports = function(grunt) {
+
+  require('grunt-task-loader')(grunt)
```
Comment (mnd-hound) [inline Gruntfile.js]: Missing semicolon.

File: Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    sass: {
+      dist: {
+        options: {
+          style: 'expanded',
+          update: true,
+          trace: true,
+          loadPath: ['bower_components']
+        },
+        files: {
+          '<%= PUBLIC %>/mnd-bootstrap-dev.css': '<%= SRC %>/mnd-bootstrap-dev.scss',
```
Comment (mnd-hound) [inline Gruntfile.js]: Line is too long.

File: Gruntfile.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    webpack: {
+      dist: {
+        entry: './hologram/mnd.js',
+        output: {
+          path: '<%= PUBLIC %>',
+          filename: 'mnd.js'
+        },
+        module: {
+          loaders: [
+            {test: /\.coffee$/, exclude: /node_modules/, loaders: ['jsx-loader', 'coffee-loader']},
```
Comment (mnd-hound) [inline Gruntfile.js]: Line is too long.

File: Gruntfile.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+            {test: /\.js$/, exclude: /node_modules|hologram/, loader: 'babel-loader'}
```
Comment (mnd-hound) [inline Gruntfile.js]: Line is too long.

File: Gruntfile.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    hologram: {
+      dist: {
+        options: {
+          config: 'hologram/hologram_config.yml'
+        }
+      }
+    },
+    watch: {
+      js: {
+        files: ['<%= SRC %>/**/*.js', '<%= SRC %>/**/*.coffee', 'hologram/mnd.js'],
```
Comment (mnd-hound) [inline Gruntfile.js]: Line is too long.

File: Gruntfile.js [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+      options: {
+        base: '<%= PUBLIC %>'
+      },
+      dist: {
+        src: ['**']
+      }
+    }
+  });
+
+  grunt.registerTask('build', ['copy', 'sass', 'webpack', 'hologram'])
```
Comment (mnd-hound) [inline Gruntfile.js]: Missing semicolon.

File: Gruntfile.js [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+  grunt.registerTask('default', ['build', 'connect', 'watch'])
```
Comment (mnd-hound) [inline Gruntfile.js]: Missing semicolon.

File: Gruntfile.js [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+  grunt.registerTask('deploy', ['build', 'gh-pages']
+}
```
Comment (mnd-hound) [inline Gruntfile.js]: Expected ')' and instead saw '}'.<br>Missing semicolon.

File: Gruntfile.js [hunk 9]
```diff
@@ -0,0 +1,119 @@
+module.exports = function(grunt) {
```
Comment (mnd-hound) [inline Gruntfile.js]: Unmatched '{'.

File: Gruntfile.js [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+        module: {
+          loaders: [
+            {test: /\.coffee$/, exclude: /node_modules/, loaders: ['jsx-loader', 'coffee-loader']},
+            {test: /\.js$/, exclude: /node_modules|hologram/, loader: 'babel-loader'}
+          ]
+        },
+        resolve: {
+          extensions: ['', '.js', '.coffee'],
+          alias: {
+            components: __dirname + '/<%= SRC %>/react-components',
```
Comment (mnd-hound) [inline Gruntfile.js]: '__dirname' is not defined.

File: hologram/mnd.js [hunk 11]
```diff
@@ -0,0 +1,3 @@
+window.React = require('react')
```
Comment (mnd-hound) [inline hologram/mnd.js]: Missing semicolon.

File: hologram/mnd.js [hunk 12]
```diff
@@ -0,0 +1,3 @@
...
+window.DatePicker = require('components/datepicker')
```
Comment (mnd-hound) [inline hologram/mnd.js]: Missing semicolon.

File: hologram/mnd.js [hunk 13]
```diff
@@ -0,0 +1,3 @@
...
+window.Dropdown = require('components/dropdown')
```
Comment (mnd-hound) [inline hologram/mnd.js]: Missing semicolon.

File: Gruntfile.js (hunk 2 above)
Comment (b2l) [inline Gruntfile.js]: Ignoring you
It's a config file and it will be hard to read if it's on two lines

File: Gruntfile.js [hunk 14]
```diff
@@ -0,0 +1,131 @@
...
+      dist: {
+        src: ['**']
+      }
+    }
+  });
+
+  grunt.registerTask('build', ['copy', 'sass', 'webpack', 'hologram']);
+  grunt.registerTask('default', ['build', 'connect', 'watch']);
+  grunt.registerTask('deploy', ['build', 'gh-pages']);
+}
```
Comment (mnd-hound) [inline Gruntfile.js]: Missing semicolon.

File: Gruntfile.js [hunk 15]
```diff
@@ -0,0 +1,170 @@
...
+    }
+  });
+
+  grunt.registerTask('build', ['copy', 'sass', 'webpack', 'hologram']);
+  grunt.registerTask('default', ['build', 'connect', 'watch']);
+
+  grunt.registerMultiTask('pack', 'package file for npm', function() {
+    var entryPoint = 'module.exports = {\n';
+    var entries = [];
+    var exports = {}
```
Comment (mnd-hound) [inline Gruntfile.js]: Missing semicolon.<br>'exports' is defined but never used.

File: Gruntfile.js [hunk 16]
```diff
@@ -0,0 +1,170 @@
...
+      dest = path.join(this.data.dest, filename + '.js');
+      if (extension === 'js') {
+        code = babel.transformFileSync(file, {optional: ['runtime']}).code;
+        grunt.file.write(dest, code);
+      } else if (extension === 'coffee') {
+        coffeeCode = grunt.file.read(file);
+        jsCode = coffee.compile(coffeeCode);
+        grunt.file.write(dest, jsCode);
+      }
+      grunt.log.writeln('Compile ' + file + ' to ' + dest)
```
Comment (mnd-hound) [inline Gruntfile.js]: Missing semicolon.<br>'dest' is not defined.

File: Gruntfile.js [hunk 17]
```diff
@@ -0,0 +1,170 @@
...
+    }, this);
+
+    entryPoint += entries.join(',\n');
+    entryPoint += '\n};'
```
Comment (mnd-hound) [inline Gruntfile.js]: Missing semicolon.

File: Gruntfile.js [hunk 18]
```diff
@@ -0,0 +1,170 @@
...
+  grunt.registerTask('build', ['copy', 'sass', 'webpack', 'hologram']);
+  grunt.registerTask('default', ['build', 'connect', 'watch']);
+
+  grunt.registerMultiTask('pack', 'package file for npm', function() {
+    var entryPoint = 'module.exports = {\n';
+    var entries = [];
+    var exports = {}
+    var files = grunt.file.expand(this.data.src);
+    files.forEach( function(file, index) {
+      extension = path.extname(file);
```
Comment (mnd-hound) [inline Gruntfile.js]: 'extension' is not defined.

File: Gruntfile.js [hunk 19]
```diff
@@ -0,0 +1,170 @@
...
+      filename = path.basename(file, extension);
```
Comment (mnd-hound) [inline Gruntfile.js]: 'filename' is not defined.<br>'extension' is not defined.

File: Gruntfile.js [hunk 20]
```diff
@@ -0,0 +1,170 @@
...
+      var className = changeCase.pascalCase(filename);
```
Comment (mnd-hound) [inline Gruntfile.js]: 'filename' is not defined.

File: Gruntfile.js [hunk 21]
```diff
@@ -0,0 +1,170 @@
...
+      entries.push('  "' + className + '": require("./lib/' + filename + '")');
```
Comment (mnd-hound) [inline Gruntfile.js]: 'filename' is not defined.

File: Gruntfile.js [hunk 22]
```diff
@@ -0,0 +1,170 @@
...
+      dest = path.join(this.data.dest, filename + '.js');
```
Comment (mnd-hound) [inline Gruntfile.js]: 'dest' is not defined.<br>'filename' is not defined.

File: Gruntfile.js [hunk 23]
```diff
@@ -0,0 +1,170 @@
...
+      if (extension === 'js') {
```
Comment (mnd-hound) [inline Gruntfile.js]: 'extension' is not defined.

File: Gruntfile.js [hunk 24]
```diff
@@ -0,0 +1,170 @@
...
+        code = babel.transformFileSync(file, {optional: ['runtime']}).code;
```
Comment (mnd-hound) [inline Gruntfile.js]: 'code' is not defined.

File: Gruntfile.js [hunk 25]
```diff
@@ -0,0 +1,170 @@
...
+        grunt.file.write(dest, code);
```
Comment (mnd-hound) [inline Gruntfile.js]: 'dest' is not defined.<br>'code' is not defined.

File: Gruntfile.js [hunk 26]
```diff
@@ -0,0 +1,170 @@
...
+      } else if (extension === 'coffee') {
```
Comment (mnd-hound) [inline Gruntfile.js]: 'extension' is not defined.

File: Gruntfile.js [hunk 27]
```diff
@@ -0,0 +1,170 @@
...
+        coffeeCode = grunt.file.read(file);
```
Comment (mnd-hound) [inline Gruntfile.js]: 'coffeeCode' is not defined.

File: Gruntfile.js [hunk 28]
```diff
@@ -0,0 +1,170 @@
...
+        jsCode = coffee.compile(coffeeCode);
```
Comment (mnd-hound) [inline Gruntfile.js]: 'jsCode' is not defined.<br>'coffeeCode' is not defined.

File: Gruntfile.js [hunk 29]
```diff
@@ -0,0 +1,170 @@
...
+        grunt.file.write(dest, jsCode);
```
Comment (mnd-hound) [inline Gruntfile.js]: 'dest' is not defined.<br>'jsCode' is not defined.

File: Gruntfile.js [hunk 30]
```diff
@@ -0,0 +1,170 @@
...
+
+  grunt.registerTask('build', ['copy', 'sass', 'webpack', 'hologram']);
+  grunt.registerTask('default', ['build', 'connect', 'watch']);
+
+  grunt.registerMultiTask('pack', 'package file for npm', function() {
+    var entryPoint = 'module.exports = {\n';
+    var entries = [];
+    var exports = {}
+    var files = grunt.file.expand(this.data.src);
+    files.forEach( function(file, index) {
```
Comment (mnd-hound) [inline Gruntfile.js]: 'index' is defined but never used.

File: dist/lib/datepicker.js [hunk 31]
```diff
@@ -0,0 +1,207 @@
...
+  minDate: new Date(2014, 1, 1),
+  maxDate: new Date(2016, 2, 2),
+  onChange: function(date) { console.log(date) },
+  disabled: false
+}
+---
+<DatePicker {...props} />
+```
+*/
+"use strict";
```
Comment (mnd-hound) [inline dist/lib/datepicker.js]: Use the function form of "use strict".

File: dist/lib/datepicker.js [hunk 32]
```diff
@@ -0,0 +1,207 @@
...
+
+var _inherits = require("babel-runtime/helpers/inherits")["default"];
+
+var _get = require("babel-runtime/helpers/get")["default"];
+
+var _createClass = require("babel-runtime/helpers/create-class")["default"];
+
+var _classCallCheck = require("babel-runtime/helpers/class-call-check")["default"];
```
Comment (mnd-hound) [inline dist/lib/datepicker.js]: Line is too long.

File: dist/lib/dropdown.js [hunk 33]
```diff
@@ -0,0 +1,72 @@
...
+---
+<Dropdown {...props}>
+  <a>Hello</a>
+</Dropdown>
+```
+ */
+
+(function() {
+  var Dropdown, React,
+    extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
```
Comment (mnd-hound) [inline dist/lib/dropdown.js]: Line is too long.<br>Expected '{' and instead saw 'child'.<br>A constructor name should start with an uppercase letter.

File: dist/lib/dropdown.js [hunk 34]
```diff
@@ -0,0 +1,72 @@
...
+  React = require("react");
+
+  Dropdown = (function(superClass) {
+    extend(Dropdown, superClass);
+
+    Dropdown.propTypes = {
+      label: React.PropTypes.string.isRequired
+    };
+
+    function Dropdown(props) {
```
Comment (mnd-hound) [inline dist/lib/dropdown.js]: 'Dropdown' is already defined.

File: dist/lib/dropdown.js [hunk 35]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    Dropdown.prototype.toggleDropdown = function() {
+      return this.setState({
+        open: !this.state.open
+      });
+    };
+
+    Dropdown.prototype.renderDropdown = function() {
+      return <div className="dropdown-content">
```
Comment (mnd-hound) [inline dist/lib/dropdown.js]: Expected an identifier and instead saw '<'.<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.

File: dist/lib/dropdown.js [hunk 36]
```diff
@@ -0,0 +1,72 @@
...
+      {this.props.children}
```
Comment (mnd-hound) [inline dist/lib/dropdown.js]: Expected ':' and instead saw '.'.

File: dist/lib/dropdown.js [hunk 37]
```diff
@@ -0,0 +1,72 @@
...
+    </div>;
```
Comment (mnd-hound) [inline dist/lib/dropdown.js]: Unclosed regular expression.<br>Unrecoverable syntax error. (71% scanned).

File: grunt/pack.js [hunk 38]
```diff
@@ -0,0 +1,36 @@
+var changeCase = require('change-case');
+var path = require('path');
+var babel = require('babel-core');
+var coffee = require('coffee-script');
+
+module.exports = function() {
+  var filename = extension = componentName = null;
```
Comment (mnd-hound) [inline grunt/pack.js]: You might be leaking a variable (extension) here.<br>'extension' is not defined.<br>'componentName' is not defined.

File: grunt/pack.js [hunk 39]
```diff
@@ -0,0 +1,36 @@
...
+  files.forEach(function(file) {
+    extension = path.extname(file);
+    filename = path.basename(file, extension);
+    componentName = changeCase.pascalCase(filename);
+    components.push({
+      name: componentName,
+      fileName: './lib/' + filename
+    });
+    dest = path.join(this.data.dest, filename + '.js');
+    var code = null
```
Comment (mnd-hound) [inline grunt/pack.js]: Missing semicolon.

File: grunt/pack.js [hunk 40]
```diff
@@ -0,0 +1,36 @@
...
+    grunt.log.writeln('Compile ' + file + ' to ' + dest);
+  }, this);
+
+  var entryPoint = 'module.exports = {\n';
+  entryPoint += components.map( function(component) {
+    return '  "' + component.name + '": require("' + component.fileName + '")';
+  }).join(',\n');
+  entryPoint += '\n};';
+  grunt.file.write(this.data.manifest, entryPoint);
+}
```
Comment (mnd-hound) [inline grunt/pack.js]: Missing semicolon.

File: grunt/pack.js [hunk 41]
```diff
@@ -0,0 +1,36 @@
+var changeCase = require('change-case');
+var path = require('path');
+var babel = require('babel-core');
+var coffee = require('coffee-script');
+
+module.exports = function() {
+  var filename = extension = componentName = null;
+  var components = [];
+  var files = grunt.file.expand(this.data.src);
```
Comment (mnd-hound) [inline grunt/pack.js]: 'grunt' is not defined.

File: grunt/pack.js [hunk 42]
```diff
@@ -0,0 +1,36 @@
...
+  files.forEach(function(file) {
+    extension = path.extname(file);
```
Comment (mnd-hound) [inline grunt/pack.js]: 'extension' is not defined.

File: grunt/pack.js [hunk 43]
```diff
@@ -0,0 +1,36 @@
...
+    filename = path.basename(file, extension);
```
Comment (mnd-hound) [inline grunt/pack.js]: 'extension' is not defined.

File: grunt/pack.js [hunk 44]
```diff
@@ -0,0 +1,36 @@
...
+    componentName = changeCase.pascalCase(filename);
```
Comment (mnd-hound) [inline grunt/pack.js]: 'componentName' is not defined.

File: grunt/pack.js [hunk 45]
```diff
@@ -0,0 +1,36 @@
...
+    components.push({
+      name: componentName,
```
Comment (mnd-hound) [inline grunt/pack.js]: 'componentName' is not defined.

File: grunt/pack.js [hunk 46]
```diff
@@ -0,0 +1,36 @@
...
+      fileName: './lib/' + filename
+    });
+    dest = path.join(this.data.dest, filename + '.js');
```
Comment (mnd-hound) [inline grunt/pack.js]: 'dest' is not defined.

File: grunt/pack.js [hunk 47]
```diff
@@ -0,0 +1,36 @@
...
+    var code = null
+    if (extension === '.js') {
```
Comment (mnd-hound) [inline grunt/pack.js]: 'extension' is not defined.

File: grunt/pack.js [hunk 48]
```diff
@@ -0,0 +1,36 @@
...
+      code = babel.transformFileSync(file, {optional: ['runtime']}).code;
+    } else if (extension === '.coffee') {
```
Comment (mnd-hound) [inline grunt/pack.js]: 'extension' is not defined.

File: grunt/pack.js [hunk 49]
```diff
@@ -0,0 +1,36 @@
...
+      var coffeeCode = grunt.file.read(file);
```
Comment (mnd-hound) [inline grunt/pack.js]: 'grunt' is not defined.

File: grunt/pack.js [hunk 50]
```diff
@@ -0,0 +1,36 @@
...
+      code = coffee.compile(coffeeCode);
+    }
+    grunt.file.write(dest, code);
```
Comment (mnd-hound) [inline grunt/pack.js]: 'grunt' is not defined.<br>'dest' is not defined.

File: dist/lib/date_picker.js [hunk 51]
```diff
@@ -0,0 +1,207 @@
...
+  minDate: new Date(2014, 1, 1),
+  maxDate: new Date(2016, 2, 2),
+  onChange: function(date) { console.log(date) },
+  disabled: false
+}
+---
+<DatePicker {...props} />
+```
+*/
+"use strict";
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Use the function form of "use strict".

File: dist/lib/date_picker.js [hunk 52]
```diff
@@ -0,0 +1,207 @@
...
+
+var _inherits = require("babel-runtime/helpers/inherits")["default"];
+
+var _get = require("babel-runtime/helpers/get")["default"];
+
+var _createClass = require("babel-runtime/helpers/create-class")["default"];
+
+var _classCallCheck = require("babel-runtime/helpers/class-call-check")["default"];
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.

File: dist/lib/date_picker.js [hunk 53]
```diff
@@ -0,0 +1,207 @@
...
+
+var _Object$defineProperty = require("babel-runtime/core-js/object/define-property")["default"];
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.

File: dist/lib/date_picker.js [hunk 54]
```diff
@@ -0,0 +1,207 @@
...
+
+var _interopRequireDefault = require("babel-runtime/helpers/interop-require-default")["default"];
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.

File: dist/lib/date_picker.js [hunk 55]
```diff
@@ -0,0 +1,207 @@
...
+
+var _classnames2 = _interopRequireDefault(_classnames);
+
+var _reactWidgets = require("react-widgets");
+
+var DatePicker = (function (_React$Component) {
+  function DatePicker() {
+    _classCallCheck(this, DatePicker);
+
+    _get(Object.getPrototypeOf(DatePicker.prototype), "constructor", this).call(this);
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.

File: dist/lib/date_picker.js [hunk 56]
```diff
@@ -0,0 +1,207 @@
...
+    key: "hideCalendar",
+    value: function hideCalendar() {
+      // if (this.removeClickOutside)
+      //   this.removeClickOutside()
+      this.setState({ isOpen: false });
+    }
+  }, {
+    key: "validate",
+    value: function validate(date) {
+      var d = (0, _moment2["default"])(date);
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Bad invocation.

File: dist/lib/date_picker.js [hunk 57]
```diff
@@ -0,0 +1,207 @@
...
+      var isValid = d.isValid();
+      if (this.props.min) isValid = isValid && d.isAfter((0, _moment2["default"])(this.props.min), "day");
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.<br>Expected '{' and instead saw 'isValid'.<br>Bad invocation.

File: dist/lib/date_picker.js [hunk 58]
```diff
@@ -0,0 +1,207 @@
...
+      if (this.props.max) isValid = isValid && d.isBefore((0, _moment2["default"])(this.props.max), "day");
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.<br>Expected '{' and instead saw 'isValid'.<br>Bad invocation.

File: dist/lib/date_picker.js [hunk 59]
```diff
@@ -0,0 +1,207 @@
...
+      return isValid;
+    }
+  }, {
+    key: "setDate",
+    value: function setDate(date) {
+      this.hideCalendar();
+      if (this.props.onChange) this.props.onChange(date);
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Expected '{' and instead saw 'this'.

File: dist/lib/date_picker.js [hunk 60]
```diff
@@ -0,0 +1,207 @@
...
+    }
+  }, {
+    key: "onDateChange",
+    value: function onDateChange(e) {
+      e.preventDefault();
+      var input = e.target.dateInput;
+      var date = new Date(Date.parse(input.value));
+      if (!this.validate(date)) return;
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Expected '{' and instead saw 'return'.

File: dist/lib/date_picker.js [hunk 61]
```diff
@@ -0,0 +1,207 @@
...
+      this.setDate(date);
+    }
+  }, {
+    key: "handleKeyDown",
+    value: function handleKeyDown(e) {
+      if (e.keyCode === 27) this.hideCalendar();
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Expected '{' and instead saw 'this'.

File: dist/lib/date_picker.js [hunk 62]
```diff
@@ -0,0 +1,207 @@
...
+    }
+  }, {
+    key: "renderLabel",
+    value: function renderLabel() {
+      if (!this.props.label) return;
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Expected '{' and instead saw 'return'.

File: dist/lib/date_picker.js [hunk 63]
```diff
@@ -0,0 +1,207 @@
...
+      return _react2["default"].createElement(
+        "label",
+        { className: "date-picker-label" },
+        this.props.label
+      );
+    }
+  }, {
+    key: "renderForm",
+    value: function renderForm() {
+      var date = (0, _moment2["default"])(this.props.value).format("YYYY-MM-DD");
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.<br>Bad invocation.

File: dist/lib/date_picker.js [hunk 64]
```diff
@@ -0,0 +1,207 @@
...
+      var inputProps = {
+        className: "date-picker-field-input",
+        defaultValue: date,
+        onKeyDown: this.handleKeyDown.bind(this),
+        onFocus: this.showCalendar.bind(this),
+        type: "text",
+        name: "dateInput"
+      };
+
+      if (this.props.disabled) inputProps.disabled = true;
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Expected '{' and instead saw 'inputProps'.

File: dist/lib/date_picker.js [hunk 65]
```diff
@@ -0,0 +1,207 @@
...
+      return _react2["default"].createElement(
+        "form",
+        formProps,
+        _react2["default"].createElement("input", inputProps)
+      );
+    }
+  }, {
+    key: "renderCalendar",
+    value: function renderCalendar() {
+      if (!this.state.isOpen) return;
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Expected '{' and instead saw 'return'.

File: dist/lib/date_picker.js [hunk 66]
```diff
@@ -0,0 +1,207 @@
...
+
+      var wrapperCN = (0, _classnames2["default"])("date-picker-calendar", { "hidden": !this.state.isOpen });
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.<br>Bad invocation.

File: dist/lib/date_picker.js [hunk 67]
```diff
@@ -0,0 +1,207 @@
...
+        "div",
+        { className: wrapperCN },
+        _react2["default"].createElement(
+          "div",
+          { className: "popover bottom fade in" },
+          _react2["default"].createElement("div", { className: "arrow" }),
+          _react2["default"].createElement(
+            "div",
+            { className: "popover-content" },
+            _react2["default"].createElement(_reactWidgets.Calendar, calendarProps)
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.

File: dist/lib/date_picker.js [hunk 68]
```diff
@@ -0,0 +1,207 @@
...
+          )
+        )
+      );
+    }
+  }, {
+    key: "render",
+    value: function render() {
+      var label = null;
+      if (this.props.label) label = this.renderLabel();
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Expected '{' and instead saw 'label'.

File: dist/lib/date_picker.js [hunk 69]
```diff
@@ -0,0 +1,207 @@
...
+
+      var form = this.renderForm();
+      var calendar = this.renderCalendar();
+
+      return _react2["default"].createElement(
+        "div",
+        { className: "date-picker", "aria-role": "group", "aria-label": "datepicker" },
```
Comment (mnd-hound) [inline dist/lib/date_picker.js]: Line is too long.

**Pull request closed without merging**